src/components/learner-credit-management/Budgetcard-V3.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable */
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import dayjs from 'dayjs';
+import {
+    Card,
+    Button,
+    Stack,
+    Row,
+    Col,
+} from '@edx/paragon';
+
+import { BUDGET_STATUSES, ROUTE_NAMES } from '../EnterpriseApp/data/constants';
+import { getBudgetStatus } from './data/utils';
+
+const SubBudgetCard = ({
+    id,
+    start,
+    end,
+    available,
+    spent,
+    displayName,
+    enterpriseSlug,
+}) => {
+    const formattedStartDate = dayjs(start).format('MMMM D, YYYY');
```

Comment:
can we write a utility which will return us the formatted date? This way we will have the same format always.

Reply:
> can we write a utility which will return us the formatted date? This way we will have the same format always.

Actually, we are going to update the date term in our next sprint tickets. So this thing will be handled there efficiently